humans/taylor_richard/details.json

```diff
@@ -4,12 +4,12 @@
   "role": [
     "Speaker"
   ],
-  "twitter": "",
+  "twitter": "https://twitter.com/rightincode",
   "facebook": "",
-  "linkedin": "",
-  "personalWebsite": "",
-  "companyWebsite": "",
-  "companyName": "rightincode",
+  "linkedin": "https://www.linkedin.com/in/rightincode/",
+  "personalWebsite": "http://www.rightincode.com/",
+  "companyWebsite": "http://logicaladvantage.com/",
+  "companyName": "Logical Advantage",
   "firstName": "Richard",
   "lastName": "Taylor"
 }
```

Comment:
Just a few things. Your Twitter should just be the handle (no `@`) and escape the slashes in the urls like `http:\/\/`.